src/main/java/com/gamingmesh/jobs/container/Job.java

```diff
@@ -643,17 +644,25 @@ public boolean isWorldBlackListed(Block block) {
     }
 
     public boolean isWorldBlackListed(Block block, Entity ent) {
-	if (worldBlacklist.isEmpty())
-	    return false;
+        if (worldBlacklist.isEmpty())
+            return isReversedBlacklistFunctionality();
 
-	if (block != null && worldBlacklist.contains(block.getWorld().getName()))
-	    return true;
+        if (block != null)
+            return worldBlacklist.contains(block.getWorld().getName()) != isReversedBlacklistFunctionality();
 
-	return ent != null && worldBlacklist.contains(ent.getWorld().getName());
+        return ent != null && worldBlacklist.contains(ent.getWorld().getName()) != isReversedBlacklistFunctionality();

```

Comment:
Do not call directly the reversed method, instead use the field property.
```suggestion
        return ent != null && worldBlacklist.contains(ent.getWorld().getName()) != reversedWorldBlacklist;
```
(You need to apply the first review before committing this)